docs/usage.md

```diff
@@ -55,10 +85,12 @@ The Image Viewer widget allows users to display images or diagrams that are rele
 
 The primary purpose of the *Global Visibility* playbook collection is to gather data from individual sites and transform it into a format that is easily consumable by the widgets. By centralizing data within the "Global Visibility" module, the solution pack's widgets can efficiently retrieve and display information, providing users with a unified view of critical metrics and records across sites.
 
-## Global Visibility: The Module
+## Global Visibility Data: The Module
 
 The **Global Visibility** module acts as a centralized repository for the aggregated data. It ensures that the widgets have access to a consistent and up-to-date dataset for visualization. The module provides an interface through which widgets can query and retrieve data based on the selected site or other relevant parameters.
 
+![Global Visibility Scenario](./res/global-visibility-module.png)
+
 ## Global Visibility: The Widgets
 
 The widgets within the solution pack's dashboard, such as the Record Summary Card, Record Summary Tile, Funnel Chart, Top-X, rely on the "Global Visibility" module as their primary data source. The playbooks' data collection and transformation process directly influences the accuracy and comprehensiveness of data displayed by these widgets.
```

Comment:
The widgets within the solution pack's dashboard, such as the _Record Summary Card_, _Record Summary Tile_, _Funnel Chart_, and _Top-X_ rely on the **Global Visibility** module as their primary data source. The playbooks' data collection and transformation process directly influences the accuracy and comprehensiveness of data displayed by these widgets.